gravitee-am-repository/gravitee-am-repository-tests/src/test/java/io/gravitee/am/repository/oauth2/api/AccessTokenRepositoryTest.java

```diff
@@ -65,6 +64,22 @@ public void shouldFindToken() {
         observer.assertNoErrors();
     }
 
+    @Test
+    public void shouldCreateWithLongClientId() {
+        AccessToken token = new AccessToken();
```

Comment:
**todo**: sorry to bother you but could you please add same test in all impacted tables:

- devices (client)
- scope_approvals: (client_id)
- access_tokens : (client)
- refresh_tokens: (client)
- authorization_codes: (client_id)
- pushed_authorization_requests: (client)
- ciba_auth_requests: (client)